db/db_impl.h

```diff
@@ -561,6 +561,13 @@ class DBImpl : public DB {
 
   const SnapshotList& snapshots() const { return snapshots_; }
 
+  const std::vector<SequenceNumber> CopySnapshots(
+      SequenceNumber* oldest_write_conflict_snapshot = nullptr,
+      const SequenceNumber& max_seq = kMaxSequenceNumber) const {
```

Comment:
Do we need the default values here? I don't see non-default arguments passed in.